src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/InputMultilineTryWithResources.java

```diff
@@ -0,0 +1,64 @@
+package com.puppycrawl.tools.checkstyle.checks.indentation; //indent:0 exp:0
+
+import java.io.BufferedWriter; //indent:0 exp:0
+import java.io.IOException; //indent:0 exp:0
+import java.nio.charset.Charset; //indent:0 exp:0
+import java.nio.charset.MalformedInputException; //indent:0 exp:0
+import java.nio.charset.StandardCharsets; //indent:0 exp:0
+import java.nio.file.Files; //indent:0 exp:0
+import java.nio.file.Path; //indent:0 exp:0
+import java.nio.file.Paths; //indent:0 exp:0
+import java.util.zip.ZipFile; //indent:0 exp:0
+
+public final class InputMultilineTryWithResources { //indent:0 exp:0
+
+    private InputMultilineTryWithResources() { //indent:4 exp:4
+
+    } //indent:4 exp:4
+
+    static void fooMethod(String zipFileName) throws IOException { //indent:4 exp:4
+
+        Charset charset = StandardCharsets.US_ASCII; //indent:8 exp:8
+        Path outputFilePath = Paths.get(zipFileName); //indent:8 exp:8
+
+        try ( //indent:8 exp:8
+BufferedWriter writer = Files.newBufferedWriter(outputFilePath, charset); //indent:0 exp:12 warn
+            ZipFile zf = new ZipFile(zipFileName) //indent:12 exp:12
```

Comment:
I don't see any examples where 2nd variable is lower or higher than expected.
Please add them.